db/app.js

```diff
@@ -1,5 +1,14 @@
 const express = require('express');
-const {} = require(`${__dirname}/controller.js`);
+const {getTopics} = require(`${__dirname}/controller.js`);
 
 const app = express()
-app.use(express.json());
\ No newline at end of file
+app.use(express.json());
+
+app.get('/api/topics', getTopics)
+
+app.use((error, request, response, next) => {
+
+    response.status(404).send({message: 'Something went wrong'})
```

Comment:
Please complete the testing for this. Remember though TDD is test DRIVEN development 😄